### ISSUE TITLE
Tux pirate

### ISSUE THREAD

**Issue by JohannCahier** [PoggoAgency/Tux-Collection#5]
Il lui faut une jambe de bois ! (c) Jon
Je valide l'idée, pour le "coat" je l'habillerait bien "Albator"

(avec un trident sur la tête ?)

**Comment by JohannCahier** [comment]
Tricorne **

**Comment by JohannCahier** [comment]
Avec la tête de mort et deux sabres en dessous

**Comment by PoggoAgency** [comment]
C'est une belle idée, par contre ce coup-ci il nous faudrait le GO de Fulup, car ça représente pas mal d'heures et que nous en avons encore 7 à faire dans la TODOLIST :)

**Comment by JohannCahier** [comment]
On a le GO !!

Fulup le veut avec un genre de bandana rouge et un couteau entre les dents (le bec ?)
Je te forwarde la "consigne" par mail.

**Comment by JohannCahier** [comment]
Stéphane le veut**

**Comment by JohannCahier** [comment]
A IoT, on a pas sd'jambe de bois, on a des nouilles mais ça ne s'voit psa !

**Comment by kooltux** [comment]
Oui, un pirate c'est sympa, mais comme vu par mail, les Tuxs sont des gentils.... Donc pas de couteau/sabre entre les dents ! Et @PoggoAgency, ce n'est pas une priorité.
